src/html/HTMLWriter.jl

```diff
@@ -2393,16 +2393,24 @@ function domify(dctx::DCtx, node::Node, a::MarkdownAST.Admonition)
             # apply a class
             isempty(cat_sanitized) ? "" : ".is-category-$(cat_sanitized)"
         end
-
+    node_repr = sprint(io -> show(io, node))
+    content_hash = bytes2hex(SHA.sha1(node_repr))[1:8]
```

Comment:
Unfortunately this has introduced a *severe* performance regressions. For the Oscar.jl package, building its manual went from ~40 seconds to ~680 seconds on my machine. It spends all that time computing SHA1 checksums it then doesn't need...

Reply:
Rats. Do you have any sense of whether it's the SHA calculation that's slow, or the rendering of the block (i.e. the `sprint` line)?

Reply:
I've interrupted this via ctrl-c a few (~5?) times and each time it was in the middle of a SHA computation. Not a proof yet but quite suggestive

Reply:
We don't need crypto safety here. Perhaps a weaker but faster hash suffices. Maybe even CRC32?

Reply:
Isn't Base.hash of the string sufficient?